RAP/rap-faq.qmd

```diff
@@ -197,7 +197,7 @@ In short, a single repository should be used for all releases of your publicatio
 
 Yes, you should have some plain text documentation, this is a part of the guidance on RAP and a part of the 'documentation' step to 'good' practice.
 
-The README in your repository is the place for traditional 'desk notes' and text explanations of bits of the process and context required, we have [guidance on writing READMEs on the statistics production website](rap.html#Writing_a_README_file).
+The README in your repository is the place for traditional 'desk notes' and text explanations of bits of the process and context required, we have [guidance on writing READMEs on the statistics production website](rap-statistics.html#Writing_a_README_file).
```

Comment:
When I click this link it just takes me to the top of the RAP for statistics page